f/flatpak-builder/stone.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# SPDX-FileCopyrightText: © 2025 - AerynOS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : flatpak-builder
+version     : 1.4.7
+release     : 1
+homepage    : https://github.com/flatpak/flatpak-builder
+upstreams   :
+    - https://github.com/flatpak/flatpak-builder/releases/download/1.4.7/flatpak-builder-1.4.7.tar.xz : fd5bc36fe3b974395f782e6c920d8955cee168f513370c32cc800b69acd980d0
+summary     : flatpak-builder is a tool for building flatpaks from sources.
+description : |
+    flatpak-builder is a tool for building flatpaks from sources.
+license     : LGPL-2.1-only
+builddeps   :
+    - binary(appstreamcli)
+    - binary(debugedit)
+    - cmake(libxml2)
```

Comment:
For sheer consistency I'd rather this be `pkgconfig(libxml2)` instead. I generally don't use `cmake(*)` unless the package I'm building uses cmake itself. Meson uses pkgconfig for finding deps so it's better to express the builddep in the same way.